code/modules/antagonists/clockcult/clock_scriptures/scripture_scripts.dm

```diff
@@ -97,6 +97,23 @@
 	quickbind = TRUE
 	quickbind_desc = "Creates a Judicial Visor, which can smite an area, applying Belligerent and briefly stunning."
 
+/datum/clockwork_scripture/create_object/clockwork_toolset
+	descname = "clockarm"
+	name = "Clockwork Toolset"
+	desc = "Creates a autosurgeon with a implantable arm with clockwork tools"
+	invocations = list("Give me da arm")
```

Comment:
please change this

Reply:
When it’s actually said ingame it doesn’t say that it’s translates to rkt13